packages/node_modules/@ciscospark/widget-message-meet/src/components/activity-menu/index.js

```diff
@@ -26,10 +26,12 @@ export default function MessageMeetActivityMenu({
     }
   ];
   return (
-    <ActivityMenu
-      buttons={buttons}
-      onExit={onExit}
-    />
+    <div className={styles.activityMenuContainer}>
```

Comment:
There's a parent container that already has a user configurable name... this is just so we have some context in the component to grab onto with our scoped styles.